charts/kube-prometheus-stack/values.yaml

```diff
@@ -1540,6 +1540,7 @@ prometheus:
     portName: grpc
     port: 10901
     targetPort: "grpc"
+    clusterIP: ""
```

Comment:
Not sure if change of current value is wanted here, as this could break the chart for users which are upgrading.

Reply:
@monotek `spec.clusterIP` field of Service is an immutable so an error occurs when upgrading. I've fixed the default value to "None". Thanks!